pypit/scripts/test_telluric_correction.py

```diff
@@ -0,0 +1,194 @@
+import sys
+from copy import deepcopy
+import numpy as np
+from numpy.polynomial.chebyshev import chebfit, chebval
+#import scipy.stats as st
+from scipy.optimize import fmin_cg, minimize
+import matplotlib.pyplot as plt
+from astropy.convolution import Gaussian1DKernel, convolve
+#import emcee
+from linetools.spectra.xspectrum1d import XSpectrum1D
+from pypit import arload
+
+
+# Data preperation functions:
+#   get_lrisr_resolution
+#   get_data
+#   get_kernel
+#   get_transimission
+
+# Fitting functions:
+#   get_template
+#   lnlike_tf
+#   opposite_lnlike_tf
+#   get_fscale
+#   tcorrect_data
+
+def get_lrisr_resolution(wavelength):
```

Comment:
This will either need to be (mainly) instrument independent or 
that we measure R from the sky lines ourselves.